io.github.ilius.PyGlossary.yml

```diff
@@ -0,0 +1,96 @@
+  - --socket=x11
+  - --share=ipc
+  - --talk-name=org.gtk.vfs.*
+
+modules:
+  - name: lzo
+    config-opts:
+      - "--enable-shared"
+      - "--disable-static"
+    cleanup:
+      - /include
+      - /share/doc
+      - "*.la"
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: "http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz"
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+        x-checker-data:
+          type: antiya
+          project-id: 1868
+          url-pattern: http://www.oberhumer.com/opensource/lzo/download/lzo-$version.tar.gz
+
+  - name: xapian-core
+    config-opts:
+      - "--disable-sse"
+      - "--disable-backend-chert"
+      - "--disable-backend-remote"
+      - "--disable-documentation"
+    sources:
+      - type: archive
+        sha256: 196ddbb4ad10450100f0991a599e4ed944cbad92e4a6fe813be6dce160244b77
+        url: "http://mirror.download.kiwix.org/dev/kiwix-build/xapian-core-1.4.18.tar.xz"
+        x-checker-data:
+          type: antiya
+          project-id: 20193
+          url-pattern: http://mirror.download.kiwix.org/dev/kiwix-build/xapian-core-$version.tar.xz
+
+  - name: libzim
+    buildsystem: meson
+    builddir: true
+    config-opts:
+      - --wrap-mode=nodownload
+      - -Dtest_data_dir=none
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+    sources:
+      - type: git
+        url: https://github.com/openzim/libzim.git
+        tag: 8.1.0
+        commit: 9bc584edd178492f2bc8dc2a43889319980e6564
+        x-checker-data:
+          type: antiya
+          project-id: 20193
+          tag-pattern: $version
+
+  - name: pyglossary
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py build
+      - python3 setup.py install "--prefix=${FLATPAK_DEST}"
+
+      - desktop-file-edit --set-key=Exec --set-value="pyglossary --ui=gtk %U" --set-icon=${FLATPAK_ID} pkg/pyglossary.desktop
+      - install -Dm644 pkg/pyglossary.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 io.github.ilius.PyGlossary.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm644 pyglossary.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+    modules:
+      # flatpak-pip-generator --runtime org.freedesktop.Sdk//22.08 --checker-data lxml beautifulsoup4 PyICU PyYAML marisa-trie libzim python-lzo html5li
+      - python3-modules.json
+    sources:
+      - type: git
+        url: https://github.com/ilius/pyglossary.git
+        tag: 4.5.0
+        commit: 2433ff5a84aed509e11db236eb1a68e2f35ea447
+        x-checker-data:
+          type: antiya
+          project-id: 58943
+          tag-pattern: $version
+
+      - type: file
+        path: io.github.ilius.PyGlossary.metainfo.xml
+
+        # Keep until the next upstream release
+      - type: file
+        url: https://github.com/ilius/pyglossary/raw/master/res/pyglossary.svg
```

Comment:
you should pin a commit here, because if it changes on master, then the build break.